rse/grant-applicants.rst

```diff
@@ -0,0 +1,143 @@
+For grant applicants
+====================
+
+.. warning::
+
+   Grant applicants, please don't rely on this yet.  It will be
+   finalized soon.
+
+   This page is currently (2020-03) a draft for feedback.  Please send
+   corrections to the person that send this page to you, or rse-group
+   at aalto.fi
+
+
+
+If you've decided you would like to use the :doc:`research software
+engineer <index>` services in your project for a long period, you
+might want to write it directly into your grant proposals.  First off,
+your research will be better because you can you can assign
+porting/optimizing/scaling of codes, data management and open science,
```

Comment:
> `codes`

Maybe I'm old, but I just can't get used to seeing this in plural. To me, "code" and "software" are bulk terms like water and milk.

Reply:
I guess it's short for code bases or something similar. I've seen this enough that it doesn't stand out to me.